process/sync/shardblock_test.go

```diff
@@ -1405,6 +1405,7 @@ func TestBootstrap_RollBackIsEmptyCallRollBackOneBlockOkValsShouldWork(t *testin
 				//bytes represent a tx block body (strings are returns from txBlockUnit.Get which is also a stub here)
 				//copy only defined fields
 				obj = prevTxBlockBody
+				_ = obj
```

Comment:
this is masking an error
please do a deep copy

Reply:
done.